### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by itsmesarthak**
Corrected grammatical errors and made the content a bit easier to understand.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,18 +10,17 @@ to choose the right tools as their project evolves. ONNX provides an open source
...
 data types. Initially we focus on the capabilities needed for inferencing (evaluation).
 
-Caffe2, PyTorch, Microsoft Cognitive Toolkit, Apache MXNet and other tools are developing ONNX support. Enabling interoperability between different
-frameworks and streamlining the path from research to production will increase the speed of innovation in
+Caffe2, PyTorch, Microsoft Cognitive Toolkit, Apache MXNet and other tools are developing ONNX support. Enabling interoperability between different frameworks and streamlining the path from research to production will increase the speed of innovation in
 the AI community. We are an early stage and we invite the community to submit feedback and help us further
 evolve ONNX.
 
-# Use ONNX
+# Use ONNX:
```
Comment (houseroad) [inline README.md]: Can we remove `:` colon at the end of each section title?

File: README.md [hunk 2]
```diff
@@ -10,18 +10,17 @@ to choose the right tools as their project evolves. ONNX provides an open source
 It defines an extensible computation graph model, as well as definitions of built-in operators and standard
 data types. Initially we focus on the capabilities needed for inferencing (evaluation).
 
-Caffe2, PyTorch, Microsoft Cognitive Toolkit, Apache MXNet and other tools are developing ONNX support. Enabling interoperability between different
-frameworks and streamlining the path from research to production will increase the speed of innovation in
+Caffe2, PyTorch, Microsoft Cognitive Toolkit, Apache MXNet and other tools are developing ONNX support. Enabling interoperability between different frameworks and streamlining the path from research to production will increase the speed of innovation in
```
Comment (houseroad) [inline README.md]: not easy to catch what is the change here? Could you please tell me what we have changed here?

Comment (zrphercule) [reply README.md]: @houseroad I think in the old version, "frameworks" starts a new line which should not.

Comment (houseroad) [reply README.md]: In our brower, I think it will output the same results. @zrphercule Or you mean linting the doc?

**Comment by houseroad** [comment]
Not sure what's the purpose/contribution of this PR. So close.

**Pull request closed without merging**